### PR TITLE
Bump golang.org/x/crypto to 7b82a4e

### PR DESCRIPTION
**Pull request by lsm5**
Resolves: GHSA-8c26-wmh5-6g9v - CVE-2022-27191

Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

#### What type of PR is this?

> /kind other

#### What this PR does / why we need it:

Resolves: CVE-2022-27191

#### How to verify it

Buildah isn't directly affected so don't think we have an easy verifier.

#### Which issue(s) this PR fixes:

None

#### Special notes for your reviewer:

None
#### Does this PR introduce a user-facing change?

```release-note
Resolves: CVE-2022-27191
```

@nalind @TomSweeneyRedHat @rhatdan @vrothberg @flouthoc PTAL

EDIT: Include GHSA ID.

### CODE REVIEW

**Comment by nalind** [comment]
LGTM

**Review by mtrmac — COMMENTED** [review]
LGTM.

When this is on the main branch (not on a stable one), and per discussion elsewhere Dependabot will probably ever update x/crypto on its own (because the package doesn’t tag releases), wouldn’t it be nice to update all the way to the latest commit, so that we aren’t behind, at least for a few hours?

File: go.mod [hunk 1]
```diff
@@ -37,7 +37,7 @@ require (
 	github.com/stretchr/testify v1.7.1
 	github.com/syndtr/gocapability v0.0.0-20200815063812-42c35b437635
 	go.etcd.io/bbolt v1.3.6
-	golang.org/x/crypto v0.0.0-20211215153901-e495a2d5b3d3
+	golang.org/x/crypto v0.0.0-20220314234659-1baeb1ce4c0b
```
[the remark this answers is not in the document]
Comment (lsm5) [reply go.mod:40]: @mtrmac actually I just included the one that was mentioned in the cve bugzilla, but I'm cool with redoing this to use the latest commit.

**Comment by lsm5** [comment]
@nalind @mtrmac PR updated, PTAL.

**Review by flouthoc — APPROVED** [review]
LGTM
/lgtm
/hold

**Pull request merged**